lambda/src/main/resources/db/migration/V50__add_export_version.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "Consignment"
+    ADD COLUMN "ExportVersion" text
```

Comment:
Can you add a semicolon to the end please. It doesn't stop it working but it helps sometimes when I'm trying to build images for test databases.
Can you add a line break at the end of the file as well.

Reply:
🤦 , forgot about the semi-colons; been spoilt by Scala 😄